common/models/user.js

```diff
@@ -693,25 +617,51 @@ module.exports = function(User) {
    * ```
    *
    * @options {Object} verifyOptions
-   * @property {String} type Must be 'email'.
+   * @property {String} type Must be `'email'` in the current implementation.
+   * @property {Function} mailer A mailer function with a static `.send() method.
+   *  It must accept the verifyOptions object, the method'sremoting context options
+   *  object and a callback function with `(err, {email, verificationToken, uid})`
```

Comment:
I find this suspicious, are you sure that `Email.send` passes ` {email, verificationToken, uid}` to the callback? I thought it was passing the email MIME message or something similar.